### PR TITLE
Post-merge EIR WWHP Polishing

### PR DESCRIPTION
**Pull request by Myoldmopar**
Pull request overview
---------------------
A few review comments were made after the EIR WWHP was abruptly merged in.  This addresses all of them.  They were primarily doc/idd/whitespace type changes.  In addition, one comment requested fixing up the init temp used in the equation fit WWHP model.  This wasn't directly related to this work, but fixed it here anyway.  This will possibly (probably?) cause diffs in a couple files that use the equation fit heat pump model.  Otherwise this should be clean and drop in whenever CI is happy.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -70097,31 +70100,28 @@ HeatPump:WaterToWater:EIR:Heating,
...
         \note CWS = supply (leaving) hot water temperature(C)
         \note ECT = entering condenser fluid temperature(C)
    A8,  \field Electric Input to Output Ratio Modifier Function of Temperature Curve Name
-        \note Electric Input Ratio (EIR) as a function of temperature
+        \note Electric Input Ratio (EIR) modifier as a function of temperature
         \note EIR = 1/COP
         \required-field
         \type object-list
-        \object-list BiquadraticCurves
         \object-list BiVariateTables
```
Comment (mjwitte) [inline idd/Energy+.idd.in:70115]: BiVariate*Functions*

**Review by mjwitte — COMMENTED** [review]
All good except one line in the idd.

File: idd/Energy+.idd.in (hunk 1 above)
Comment (Myoldmopar) [reply idd/Energy+.idd.in:70115]: If it's OK, I'll just wait for CI to be happy with this, make this tiny change and let it be merged without re-running CI for it all again.

Comment (mjwitte) [reply idd/Energy+.idd.in:70115]: Yeah,sure.

**Review by mjwitte — APPROVED** [review]

**Comment by mjwitte** [comment]
Thanks. IDF Editor is so happy now - no bad references.

**Pull request merged**